modules/network-observability-flowcollector-api-specifications.adoc

```diff
@@ -1211,6 +1238,267 @@ Type::
 
 
 
+[cols="1,1,1",options="header"]
+|===
+| Property | Type | Description
+
+| `certFile`
+| `string`
+| `certFile` defines the path to the certificate file name within the config map or secret.
+
+| `certKey`
+| `string`
+| `certKey` defines the path to the certificate private key file name within the config map or secret. Omit when the key is not necessary.
+
+| `name`
+| `string`
+| Name of the config map or secret containing certificates.
+
+| `namespace`
+| `string`
+| Namespace of the config map or secret containing certificates. If omitted, the default is to use the same namespace as where Network Observability is deployed.
+If the namespace is different, the config map or the secret is copied so that it can be mounted as required.
+
+| `type`
+| `string`
+| Type for the certificate reference: `configmap` or `secret`.
+
+|===
+== .spec.exporters[].openTelemetry
+Description::
++
+--
+OpenTelemetry configuration, such as the IP address and port to send enriched logs or metrics to.
+--
+
+Type::
+  `object`
+
+Required::
+  - `targetHost`
+  - `targetPort`
+
+
+
+[cols="1,1,1",options="header"]
+|===
+| Property | Type | Description
+
+| `fieldsMapping`
+| `array`
+| Custom fields mapping to an OpenTelemetry conformant format.
+By default, Network Observability format proposal is used: https://github.com/rhobs/observability-data-model/blob/main/network-observability.md#format-proposal .
```

Comment:
```suggestion
By default, Network Observability format proposal is used: https://github.com/rhobs/observability-data-model/blob/main/network-observability.md#format-proposal.
```

Reply:
I think there's a space here so the period doesn't interfere with the link